### PR TITLE
Fix #953: Highfi HomeFragment

### LINKED ISSUE #953: Highfi HomeScreen
Mock: https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/d47aa8de-a7a8-4d25-b011-5baefc7b7098/HP-Home-Page-Card-1-/

https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/0dbdd000-6738-4f6b-9026-0d16737b7baa/HP-Home-Page-/

This PR tracks the highfi work of Home Screen.

### PR DESCRIPTION
**Pull request by rt4914**
## Explanation

This PR tracks the highfi work of HomeScreen.

Mocks: https://xd.adobe.com/view/e8aa4198-3940-47f9-514a-f41cc54457f6-9e9b/screen/d47aa8de-a7a8-4d25-b011-5baefc7b7098/HP-Home-Page-Card-1-
https://xd.adobe.com/view/ee9e607b-dbd6-4372-48dc-b687d32af3da-98af/screen/0dbdd000-6738-4f6b-9026-0d16737b7baa/HP-Home-Page-/

## Screenshots
Screenshot_1586323197
Screenshot_1586323200
Screenshot_1586323202
Screenshot_1586323370
Screenshot_1586323375
Screenshot_1586323378

This PR passes all accessibility tests.

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/res/layout-land/all_topics.xml [hunk 1]
```diff
@@ -9,10 +9,10 @@
       android:id="@+id/all_topics_text_view"
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
-      android:layout_marginStart="72dp"
+      android:layout_marginStart="76dp"
       android:layout_marginTop="56dp"
```
Comment (veena14cs) [inline app/src/main/res/layout-land/all_topics.xml]: Margin top is 31dp in the mock.

File: app/src/main/res/layout-land/promoted_story_card.xml [hunk 2]
```diff
@@ -11,85 +11,92 @@
...
+          android:layout_height="wrap_content"
+          android:layout_marginStart="16dp"
+          android:layout_marginTop="4dp"
+          android:layout_marginEnd="16dp"
+          android:maxLines="1"
+          android:ellipsize="end"
+          android:fontFamily="sans-serif"
+          android:text="@{viewModel.promotedStoryObservable.storyName}"
+          android:textColor="@color/oppiaPrimaryText"
+          android:textSize="14sp" />
```
Comment (veena14cs) [inline app/src/main/res/layout-land/promoted_story_card.xml:82]: Textsize is 16sp in the mock.

**Review by veena14cs — APPROVED** [review]
LGTM. Address some changes.

File: app/src/main/res/layout-land/all_topics.xml (hunk 1 above)
Comment (rt4914) [reply app/src/main/res/layout-land/all_topics.xml]: Updated to `32dp`

File: app/src/main/res/layout-land/promoted_story_card.xml (hunk 2 above)
Comment (rt4914) [reply app/src/main/res/layout-land/promoted_story_card.xml:82]: Done.

**Pull request merged**